bin/run_celery_worker

```diff
@@ -9,5 +9,5 @@ cd "$SRC_DIR"
 source vagrant/env.sh
 
 exec newrelic-admin run-program celery -A treeherder worker -c 3 \
-     -Q default,cycle_data,fetch_bugs,generate_perf_alerts,seta_analyze_failures,intermittents_commenter \
+     -Q default,cycle_data,fetch_bugs,generate_perf_alerts,seta_analyze_failures \
```

Comment:
Looks like `fetch_bugs` and `cycle_data` need to be removed from here per #4019?